### DISCUSSION TITLE
Obtaining PCBs in New Zealand

### DISCUSSION THREAD

**Discussion by TerryOz** [alanbjohnston/CubeSatSim#223] [Q&A] [answered]
I have the CubeSAT Simulator sofware running on a borrowed Raspberry Pi Zero.
The Raspberry Pi Zeros are out of stock and are not due in New Zealand until after November.
AMSAT-NA have the PCB's in stock see: https://www.amsat.org/product/amsat-cubesatsim-pcb/
but will only ship to US addresses (in big red letters).
I do have the Gerber files so I could get a set of boards from a NZ company but all the ones I've looked at so far lead back to production in China hidden in the fine print.
I also have a link to the company (in China) that makes the AMSAT-NA PCBs.
If I get a set of blank PCBs then I would need to fit a small number of 0603 surface mount components.
If I was still working, I could manage this but my eyesight has deteriorated over the past 10 years and I don't think I could manage it now.
Also sourcing the 0603 parts could be a problem.
The AMSAT-NA PCBs have the 0603 parts fitted so that would be the best choice but the question is how to get hold of them?

**Comment by Boovtech** [comment, marked as answer]
Hi Terry,

Do you know what company in China is able to ship you the boards?  If it is PCBWay, they do an excellent job.  Chinese boards are fairly common and usually pretty good.  Maybe you could get in touch with them and see if thy could preinstall the SMT parts for the band pass filter.

If that is a problem, I'd be fine with helping you to get some AMSAT boards.  

My email is nv5f@arrl.net

Please let me know if I can help.

73
Virginia
